test/utils/visualize-rule-tester.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-const {Linter} = require('eslint');
+const {Linter, SourceCodeFixer} = require('eslint/lib/linter');
```

Comment:
There is no public api to apply suggestion fix, is it okay to use this internal one?

Reply:
Yes, since it's just the tests. But you should open an issue on ESLint about exposing this.